test/data/validator.output.yaml

```diff
@@ -2,30 +2,40 @@ jobs:
   main:
   - image: node:4
     steps:
-      install: npm install
-      test: npm test
-      build: npm run build
+      - name: install
```

Comment:
Javascript, YAML, JSON, and Go all don't guarantee key ordering for hashes. We absolutely have to have an array in the JSON output, and we decided that the Yaml should be written in an unambiguous way. This also leaves the door open for adding more metadata to steps over time.